### PR TITLE
Add documentation to 'value' property of @Timed

### PR DESCRIPTION
**Pull request by renannprado**

### CODE REVIEW

File: micrometer-core/src/main/java/io/micrometer/core/annotation/Timed.java [hunk 1]
```diff
@@ -22,6 +22,9 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Inherited
 public @interface Timed {
+    /**
+     * The name of the metric
```
Comment (shakuzen) [inline micrometer-core/src/main/java/io/micrometer/core/annotation/Timed.java]: This is good. "name of the Timer" would be even more precise.

Comment (renannprado) [reply micrometer-core/src/main/java/io/micrometer/core/annotation/Timed.java]: What about "The name of the timer metric"?

Comment (shakuzen) [reply micrometer-core/src/main/java/io/micrometer/core/annotation/Timed.java]: Sounds good to me

**Comment by shakuzen** [comment]
Thank you for this change @renannprado. I have cherry-picked your commit to the 1.0.x branch and merged forward.

**Pull request closed without merging**